src/App.js

```diff
@@ -1,19 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [messages, setMessages] = useState(chatMessages);
+  const [likeCount, setLikeCount] = useState(0);
+
+  const handleLikeUpdate = (id) => {
+    setMessages(messages => {
```

Comment:
👍 The next state of `messages` depends on the previous state, so using the callback style of the `setState` method is a good call.